### PR TITLE
Code cleanup and update function naming

### LINKED ISSUE #29: Variable creation and naming improvements
All variables used by the bot are currently not sorted or follow any rules. This should be changed for the last commit before the last python bot release.

### PR DESCRIPTION
**Pull request by BakxY**
Updated function and variable names and code cleanup

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -75,20 +90,30 @@ async def map(ctx):
...
         # configurate embed message
         embedVar = Map.ConfigEmbed(message, Image)
+
         # send the message
         await ctx.reply(file=Image, embed=embedVar)
-    else:
+
+    else: # API was unreachable
+        # write to log file and cli
+        print('The API is currently nor reachable by the bot')
```
Comment (FischerTG) [inline main.py]: nor -> not

File: main.py [hunk 2]
```diff
@@ -131,16 +159,19 @@ async def stats(ctx, Player):
...
-        # print error message to cli
+        await ctx.reply(RequestDataResponse['Error'], file=discord.File('src/sad-cute.gif'))
+
+        # print error message to cli and log file
         print('Error during request to API, message from API: ' + str(RequestDataResponse))
         LogFile.WriteLog('Error during request to API, message from API: ' + str(RequestDataResponse))
     
     elif APINotReachable == True:
+        # write to log file and cli
+        print('The API is currently nor reachable by the bot')
```
Comment (FischerTG) [inline main.py]: nor -> not

File: main.py [hunk 3]
```diff
@@ -172,27 +206,32 @@ async def rank(ctx, Player):
...
+        # print error message to cli and log file
         print('Error during request to API, message from API: ' + str(RequestDataResponse))
         LogFile.WriteLog('Error during request to API, message from API: ' + str(RequestDataResponse))
 
     elif APINotReachable == True:
-        # send error message to discord channel
-        await ctx.reply('The API is currently offline')
-        await ctx.reply(file=discord.File('src/sad-cute.gif'))
+        # write to log file and cli
+        print('The API is currently nor reachable by the bot')
```
Comment (FischerTG) [inline main.py]: nor -> not

File: main.py [hunk 4]
```diff
@@ -219,6 +260,14 @@ async def status(ctx):
         
         # send the message
         await ctx.reply(file=Image, embed=embedVar)
+    
+    else:
+        # write to log file and cli
+        print('The API is currently nor reachable by the bot')
```
Comment (FischerTG) [inline main.py]: nor -> not

**Review by FischerTG — COMMENTED** [review]
small typos

File: main.py (hunk 1 above)
Comment (BakxY) [reply main.py]: Changed

File: main.py (hunk 2 above)
Comment (BakxY) [reply main.py]: Changed

File: main.py (hunk 3 above)
Comment (BakxY) [reply main.py]: Changed

File: main.py (hunk 4 above)
Comment (BakxY) [reply main.py]: Changed

**Review by FischerTG — APPROVED** [review]
LGTM

**Pull request merged**